src/data/chainDetails.json

```diff
@@ -11,7 +11,6 @@
 			"https://cloudflare-eth.com",
 			"https://ethereum.publicnode.com",
 			"https://mainnet.gateway.tenderly.co",
-			"https://rpc.blocknative.com/boost",
```

Comment:
If we don't also remove blocknative from fillChainDetails.py, then if we ever update a chain it will get added back in to the rpc list.

I think either we filter it in fillChainDetails.py or we change behaviour in getDefaultProvider to filter it out there (maybe thats better)